### PR TITLE
add reduced treatment demo case study

### PR DESCRIPTION
**Pull request by melody-shellman**
## Changes proposed in this PR:
- Add new case study strategic_Permian_demo.xlsx
- Add images for case study and results for scenarios

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my
contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.md file
   at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has
   rights to intellectual property that includes these contributions, I represent that I have
   received permission to make contributions and grant the required license on behalf of that
   employer.

### CODE REVIEW

**Review by MichaelPesce — DISMISSED** [review]

**Review by tarnold17 — CHANGES_REQUESTED** [review]
Can you adjust the line breaks in the documentation table in the Permian demo column so that it has a similar width to the other columns?

I recommend not including spaces in filenames - e.g., change "Ppipeline Shutdown.png" to "pipline_shutdown.png".

File: docs/case_studies/index.rst [hunk 1]
```diff
@@ -91,43 +108,56 @@ Comparison table
...
    * - **Completions pad**
+     - No
      - No
      - Yes
      - No
      - Yes
    * - **Treatment sites**
+     - 5 |br|
+       Non-desalination sites: R02, R04, R05 |br|
+       Desalination sites: R03, R06 |br|
```
Comment (tarnold17) [inline docs/case_studies/index.rst:158]: Why isn't R01 listed as a desal site? It is included in the schematic.

**Comment by melody-shellman** [comment]
> Can you adjust the line breaks in the documentation table in the Permian demo column so that it has a similar width to the other columns?
> 
> I recommend not including spaces in filenames - e.g., change "Ppipeline Shutdown.png" to "pipline_shutdown.png".

The naming is intentional for a new feature in the UI. If the naming matches the name of a scenario, it will autopopulate in the UI.

**Review by tarnold17 — DISMISSED** [review]
Looks good to me - one last final question I have though - do you think we should add a test to make sure that the model builds correctly using `strategic_permian_demo.xlsx`? We have that test for all of the other case studies in the repo, so my inclination is that we should add it for this one as well.

**Review by MichaelPesce — APPROVED** [review]

**Review by MichaelPesce — APPROVED** [review]

**Review by tarnold17 — APPROVED** [review]

**Pull request merged**